packages/studio-plugin/src/createStudioPlugin.ts

```diff
@@ -18,7 +18,7 @@ import { UserConfig } from "vite";
 import { STUDIO_PROCESS_ARGS_OBJ } from "./constants";
 import { startPagesDevelopmentServer } from "./startPagesDevelopmentServer";
 import LocalDataMappingManager from "./LocalDataMappingManager";
-import { execSync } from "node:child_process";
+import { execSync } from "child_process";
```

Comment:
I wasn't familiar with the `node:___` syntax, so I took a look into it--  I found out that it means that it makes sure that the package is from node core and not something else.  Just curious why you removed it

Reply:
oh hmm it's just not how we've been importing things
https://2ality.com/2021/12/node-protocol-imports.html
I'm definitely down to make an item to add an eslint rule for it!